mappings/net/minecraft/world/CollisionView.mapping

```diff
@@ -10,6 +10,8 @@ CLASS net/minecraft/class_1941 net/minecraft/world/CollisionView
 	METHOD method_22338 getExistingChunk (II)Lnet/minecraft/class_1922;
 		ARG 1 chunkX
 		ARG 2 chunkZ
+	METHOD method_30030 getBlockCollisions (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;Ljava/util/function/BiPredicate;)Ljava/util/stream/Stream;
+	METHOD method_30635 isSpaceEmpty (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;Ljava/util/function/BiPredicate;)Z
```

Comment:
Should this be named similarly to the existing `doesNotCollide`?

Reply:
Hmm, didn't notice those. I could rename `method_30635` to `doesNotCollide`, same as the others, but I don't really like that name.

Reply:
Might be worth renaming them both to like `isBoxEmpty`? I don't know if that would go out of scope for this PR, though.

Reply:
Feel free to rename the existing `doesNotCollide` to `isSpaceEmpty`.